tests/test-link-preview.md

```diff
@@ -4,20 +4,74 @@ Link previews should embed previews of the contents of a hyperlink from a messag
 
 Post location variation: 
 
-1. Post as message in center channel with RHS closed (link preview should render under message) 
```

Comment:
Updated, with "Expected" added on same line. 

I'm like 0.5/5 on having it on one-line instead of two if only because nested bullets are difficult to format, and multiple lines take up a lot of vertical space.